definitions/unix_common.yaml

```diff
@@ -0,0 +1,111 @@
+# Common Unix based artifacts
```

Comment:
I think this file should only be those things that are shared between linux and OS X.  Move the linux-only ones back into the linux file.